packages/core/src/external/surescripts/fhir-converter.ts

```diff
@@ -57,41 +50,6 @@ export async function convertBatchResponseToFhirBundles(
   return conversionBundles;
 }
 
-export async function uploadConversionBundle({
```

Comment:
Moved to `packages/core/src/external/surescripts/command/bundle/save-bundle.ts` w/o changes

Reply:
Nice!! Will follow this pattern for future updates.